build/bin.NOTICE

```diff
@@ -169,3 +166,71 @@ Apache Commons Lang (lib/org.apache.commons:commons-lang3-<VERSION>.jar)
   
   This product includes software developed at
   The Apache Software Foundation (https://www.apache.org/).
+
+Apache Commons Codec (lib/commons-codec.commons-codec-<VERSION>.jar)
+  Apache Commons Codec
+  Copyright 2002-2017 The Apache Software Foundation
+
+  src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
+  contains test data from http://aspell.net/test/orig/batch0.tab.
+  Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
+
+  ===============================================================================
+
+  The content of package org.apache.commons.codec.language.bm has been translated
+  from the original php source code available at http://stevemorse.org/phoneticinfo.htm
+  with permission from the original authors.
+  Original source copyright:
+  Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
+
+Apache Commons Logging (lib/commons-logging.commons-logging-<VERSION>.jar)
+  Apache Commons Logging
+  Copyright 2003-2014 The Apache Software Foundation
+
+Apache HttpClient (lib/org.apache.httpcomponents.httpclient-<VERSION>.jar)
+  Apache HttpClient
+  Copyright 1999-2020 The Apache Software Foundation
+
+Apache HttpCore (lib/org.apache.httpcomponents.httpcore-<VERSION>.jar)
+  Apache HttpCore
+  Copyright 2005-2020 The Apache Software Foundation
+
+Apache Log4j (lib/org.apache.logging.log4j.log4j-api-<VERSION>.jar, org.apache.logging.log4j.log4j-core-<VERSION>.jar)
+  Apache Log4j
+  Copyright 1999-2019 Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  ResolverUtil.java
+  Copyright 2005-2006 Tim Fennell
+
+  Dumbster SMTP test server

```

Comment:
The bin.NOTICE file @tuxji just merged does not list Dumbster or some of these other things from Log4j. Did Log4j change their notice?

@tuxji, it would help if you could take a scan at this. You're most faimilar with all the recent changes.

Also, it's a little hard to just diff since the format has changed. Do we want to standardize all of our license/notice files to the same format?

Reply:
Eventually I think a directory of License files, and a directory of Notices files would be better than this squashing of stuff together into a single license and notices file. It's very hard to verify that this includes everything it is supposed to, and does not include things it's not supposed to, as well as not knowing what's verbatim we're including, and what is stuff we're writing.

Reply:
Yeah, we originally just copied from what other projects do, but it should be fine to have a license directory, and is definitely easier to manage. However, I'm not sure you can do the same for NOTICES. I think those have to be in the NOTICE file.

Reply:
Aye, I took a pass at this and updated bin.LICENSE and bin.NOTICE.  I noticed that many jars don't have LICENSE or NOTICE files in them but I didn't take the extra time to visit their source repositories and check for LICENSE/NOTICE files there as well, so I don't promise that my edits are the last word.  I just want to give these files a better foundation for the 1.0.0 release and let future PRs improve them.